exercises/solutions/solution_08.py

```diff
@@ -1,26 +1,25 @@
-from exercises.solutions.resources.resource_06_12 import KiwiPage, SearchResultPage, PassengerDetailsPage
+from exercises.solutions.resources.resource_07_12 import KiwiPage, SearchResultPage, PassengerDetailsPage
 
 
 # Localized currency is retained in Passenger details
 def test_localized_currency_is_retained_in_passenger_details(page):
-    # 1. Hit the 🇬🇧 CZK button in the navigation bar at the top of the search page
-    # 1.1. Open the kiwi.com website (wait for page to load)
+    # 1. Hit the 🇬🇧 CZK (regional settings) button in the navigation bar at the top of the search page
+    # 1.1. Open the kiwi.com website and accept cookies
     kiwi_page = KiwiPage(page)
     kiwi_page.open_kiwi_website_and_accept_cookies()
 
-    # 1.2 Open the localization settings and wait for the localization modal to be displayed
+    # 1.2 Open the regional settings and wait for the localization modal to be displayed
     kiwi_page.open_regional_settings()
 
-    # 2. Set the currency in the Language and currency modal to EUR
-    currency_code = "EUR"
```

Comment:
I'd keep the `currency_code` variable to always refer to the same value (here and at step 9 below).